e2e-tests/monitoring-2-0/run

```diff
@@ -126,6 +126,12 @@ wait_for_delete "pod/${cluster}-mongos-0"
 wait_for_delete "pod/${cluster}-rs0-0"
 wait_for_delete "pod/${cluster}-cfg-0"
 
+desc 'check if services are not deleted'
+
+kubectl_bin get svc $cluster-rs0
```

Comment:
maybe use something like `wait_for_delete "svc/${cluster}-rs0"`
it should be more stable I guess

Reply:
@tplavcic I can't understand why we need to use  wait_for_delete :( The goal of this check is that services should not be deleted in case of pause.

Reply:
yep, my mistake ;)